maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -81,6 +81,7 @@ private void export(File runFile) throws Exception {
 				}
 				run.setProperty(Constants.RUNBUNDLES, runBundles);
 			}
+			run.setProperty(Constants.DEFAULT_PROP_TARGET_DIR, targetDir.getAbsolutePath());
```

Comment:
The correct thing to do is call `run.setBase(new File(targetDir, "export/"+getNamePart(runFile)))`, at the start of the try block. So any directories that Project.prepare might create are in that folder.